docs/develop/01_sdk/01_quickstart.md

```diff
@@ -4,8 +4,7 @@ title: Quickstart
 ---
 import CodeBlock from '@theme/CodeBlock';
 import SnippetBlock from '@site/src/components/SnippetBlock';
-import Tabs from '@theme/Tabs';
```

Comment:
`<Tabs>` and `<TabItem>` are used in this file. Do we not have to import those anymore?

Reply:
I guess once the component is imported once, it is always there. There was no warning whatsoever either. But I fixed that, thanks! https://github.com/KILTprotocol/docs/pull/147/commits/7aef98cc92d0acdc1243d70fc8db956147bdf67c

Reply:
ah ok. yeah not sure how the imports work on the mdx files. But doesn't hurt to import them i guess